tests/integration/components/frost-object-browser-test.js

```diff
@@ -230,4 +230,31 @@ describe(test.label, function () {
       expect(displayFilter).to.have.callCount(1)
     })
   })
+
+  describe('when user mouse enters/exists the facets area', function () {
+    const myHook = 'myHook'
+    const filterHook = 'filterHook'
+
+    beforeEach(function () {
+      this.set('myHook', myHook)
+      this.set('filterHook', filterHook)
+
+      return this.render(hbs`
+        {{frost-object-browser
+          hook=myHook
+          content=(component 'mock-content' class='mock-content')
+          controls=(component 'mock-controls' class='mock-controls')
+          filters=(component 'mock-filters' class='mock-filters' hook=filterHook)
+        }}
+      `)
+    })
+
+    it('should call the handler displayFilter', function () {
+      $hook(filterHook).css('padding-left', '10px')
```

Comment:
This type of setup is supposed to be done in the `beforeEach()` - https://github.com/ciena-frost/ember-frost-test#it-blocks

Reply:
okay, i'll move it